src/Talis/Persona/Client/Tokens.php

```diff
@@ -60,15 +69,17 @@ public function validateToken(array $params = [])
      *
      * @param string $token a token to validate explicitly, if you do not
      *      specify one the method tries to find one
-     * @param array|null $scopes specify this if you wish to validate a scoped token
-     * @param integer $cacheTTL time to live value in seconds for the certificate to stay within cache
-     * @return integer ValidationResults enum
      * @throws ScopesNotDefinedException If the JWT token doesn't include the user's scopes
-     * @throws Exception If not able to communicate with Persona to retrieve the public certificate
+     * @throws CommunicationException Cannot communicate with Persona
+     * @throws \Exception If not able to communicate with Persona to retrieve the public certificate
      */
-    protected function validateTokenUsingJWT($token, array $scopes = null, $cacheTTL = 300)
+    protected function validateTokenUsingJWT($token, array $scopes = null)
```

Comment:
Why are we removing the ability to check the TTL?

Reply:
the `$cacheTTL` attribute was related to the HTTP caching layer